docusaurus.config.js

```diff
@@ -125,6 +125,11 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: true,
```

Comment:
@Petrosdevri , Why we are disabling the theme toggling, what if the user wants to switch from light to dark and vice versa?
```suggestion
        defaultMode: 'light',
  
```

Reply:
I didn't know if you wanted that, wait to fix this.